Fix various compiler warnings

mcatanzaro opened the pull request:
#### 8f811dbdfb4eca44f17ceff99db6cbce3a965300
<pre>
Fix various compiler warnings
<a href="https://bugs.webkit.org/show_bug.cgi?id=242711">https://bugs.webkit.org/show_bug.cgi?id=242711</a>

Reviewed by Carlos Garcia Campos.

In particular, it&apos;s time to stop warning about new values of
AXNotification. These would be useful if they indicated a need to
implement something, but in practice all we wind up doing is ignoring
each one individually, which is not useful.

* Source/WebCore/accessibility/atspi/AXObjectCacheAtspi.cpp:
(WebCore::AXObjectCache::postPlatformNotification):
* Source/WebCore/html/InputType.cpp:
(WebCore::InputType::isValidValue const):
* Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:
(WebCore::WebGLRenderingContextBase::texImage3DBase):
(WebCore::WebGLRenderingContextBase::texSubImage3DBase):
* Source/WebCore/platform/graphics/ImageBuffer.cpp:
(WebCore::ImageBuffer::drawPattern):

Canonical link: <a href="https://commits.webkit.org/252451@main">https://commits.webkit.org/252451@main</a>
</pre>

twilco reviewed (approved):
LGTM for the AX changes (but I'm not a WebKit reviewer).

mcatanzaro committed:
Fix various compiler warnings
https://bugs.webkit.org/show_bug.cgi?id=242711

Reviewed by Carlos Garcia Campos.

In particular, it's time to stop warning about new values of
AXNotification. These would be useful if they indicated a need to
implement something, but in practice all we wind up doing is ignoring
each one individually, which is not useful.

* Source/WebCore/accessibility/atspi/AXObjectCacheAtspi.cpp:
(WebCore::AXObjectCache::postPlatformNotification):
* Source/WebCore/html/InputType.cpp:
(WebCore::InputType::isValidValue const):
* Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:
(WebCore::WebGLRenderingContextBase::texImage3DBase):
(WebCore::WebGLRenderingContextBase::texSubImage3DBase):
* Source/WebCore/platform/graphics/ImageBuffer.cpp:
(WebCore::ImageBuffer::drawPattern):

Canonical link: https://commits.webkit.org/252451@main
